config/jenkins_infra.ci.jenkins.io.yaml

```
@@ -143,8 +143,8 @@ controller:
                   - name: jnlp-linux-arm64
                     nodeSelector: "kubernetes.io/arch=arm64"
                     containers:
-                      - name: jnlp-arm64
-                        image: "jenkins/inbound-agent:latest-jdk17"
+                      - name: jnlp
+                        image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
```

```suggestion
                        image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04:1.42.0"
```

- https://github.com/jenkins-infra/packer-images/releases/tag/1.42.0
- [hub](https://hub.docker.com/layers/jenkinsciinfra/jenkins-agent-ubuntu-22.04/1.42.0/images/sha256-79e25345aa7e871e65e741fca0eebe1b255148a2309248fe30a12903a2c46c24?context=explore)